tv_apps/remote-control/js/section.js

```diff
@@ -68,6 +68,14 @@
       if (activeChildSection) {
         activeChildSection.handleMove(direction);
       } else {
+        if (this._handleMove) {
```

Comment:
Oh, so this is for `SettingsSection`? I wonder if we have better way to write this kind of "polymorphism"..